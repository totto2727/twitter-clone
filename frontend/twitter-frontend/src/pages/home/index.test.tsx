import renderer from 'react-test-renderer'
import Home from './index'

test('home page', () => {
  const component = renderer.create(<Home />)
  const tree = component.toJSON()
  expect(tree).toMatchSnapshot()
})
