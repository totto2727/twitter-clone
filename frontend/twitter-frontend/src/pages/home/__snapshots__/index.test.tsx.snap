// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`home page 1`] = `
<main>
  <div>
    <p>
      Home
    </p>
  </div>
  <div>
    <p>
      input
    </p>
    <p>
      navi
    </p>
  </div>
  <article>
    <div>
      <p>
        timeline
      </p>
    </div>
  </article>
</main>
`;
